### DISCUSSION TITLE
Will scores made on lazer now be uploaded in the future?

### DISCUSSION THREAD

**Discussion by TheRealZophos** [ppy/osu#17025] [Q&A] [answered]
I know scores on Lazer at this current time do not have any PP value towards the current rankings on Stable. Will scores made on lazer right now be uploaded later on when the functionality is released, or are they essentially "offline only" plays

**Comment by frenzibyte** [comment, marked as answer]
They're currently submitted on osu!(lazer) to show off your recent plays, but I don't believe they'll ever award score/accuracy/PP whether it being now or when the processors for such statistics become live, as some of the gameplay mechanics may not be considered final yet (as far as I can tell, e.g. spinner inertia).

**Reply by NotGumballer91** [reply]
Is it safe to assume those previously submitted scores will award scores when it goes live?

**Reply by peppy** [reply]
if you mean pp then no. they cannot possibly.
